### ISSUE TITLE
Adding a Disclaimer

### ISSUE THREAD

**Issue by jvandevalk** [OpenDRR/riskprofiler#56]
We need to add a disclaimer for the website when people open the website. Text to come from NRCan in English and French, and coding to be done by H7.

**Comment by phil-evans** [comment]
no problem - could you provide some further direction on how this is displayed? is it a modal window that must be dismissed to continue? is it on every page/just home page/just app pages?

**Comment by drotheram** [comment]
I think probably just the home page is fine. Is it possible to display to the user the first time they visit any RiskProfiler link? Yes, modal window that must be dismissed by the user agreeing that they understand the terms of the disclaimer should be fine.

We will provide an NRCan standard language text shortly.

@tieganh @phil-evans @jvandevalk feel free to chime in

**Comment by wongstea** [comment]
I believe we discussed in a previous meeting that unless we use cookies, the disclaimer will pop up for users every time they visit the site. It would probably be fine to have the disclaimer pop up on just the home page, but if it's on every page (we can't know which specific URL people will land on if they come to the site from an external link) that would be pretty annoying. @phil-evans not sure if there's anything we can do about that.

Once we receive the English text for the disclaimer we can mock it up. We'll need the same text in French too.

**Comment by phil-evans** [comment]
yup, tia is correct - to have it be dismissable would require a browser cookie that gets checked on every page load to see if the user has agreed to the terms.

the good news is i've just finished implementing the 'page tour' component (#58) which also uses a browser cookie for the 'don't show again' checkbox:

![Screen Shot 2022-09-08 at 10 55 51 AM](https://user-images.githubusercontent.com/18401550/189140831-dad36ff1-22a5-4de5-bf77-7039e4f50cc7.png)

so it does actually make sense to do the disclaimer the way @drotheram described - we already need a cookie for the tour element, so we might as well also use one for this disclaimer. the disclaimer itself can mention that we're using these cookies, and then when the user clicks 'i agree' the whole thing kind of sorts itself out.

**Comment by phil-evans** [comment]
here's how the disclaimer modal will look - this will show up on any page if the browser cookie isn't set with the 'agree' value:

![Screen Shot 2022-09-08 at 11 25 52 AM](https://user-images.githubusercontent.com/18401550/189148854-f07fe5b1-45c0-41f9-ac54-14661cdee903.png)

if you click 'agree and continue' the cookie gets created. i have the cookie set to last 1 day for testing purposes - when we go live it should be changed to something like a year.

**Comment by drotheram** [comment]
@JulieVdV do we have the text for this yet?
